### PR TITLE
Finish task 6

### PR DESCRIPTION
**Pull request by BrokenRecord85**

### CODE REVIEW

File: src/components/SingleReview.jsx [hunk 1]
```diff
@@ -1,13 +1,31 @@
...
+import axios from 'axios'
 
 const SingleReview = () => {
    const [review, setReview] = useState({})
    const {review_id} = useParams()
-   console.log(review_id)
+   
 
+   useEffect(() => {
+        axios.get(`https://pawel-game-api.herokuapp.com/api/reviews/${review_id}`)
```
Comment (mallionaire) [inline src/components/SingleReview.jsx]: Would be nice to see you extracting your api calls to your `utils/api.js`
Should not have a mixture of them inline in components and in a dedicated file.

File: src/components/ReviewsList.jsx [hunk 2]
```diff
@@ -30,10 +30,10 @@ const ReviewsList = ({loading, setLoading}) => {
...
             <li className='review-card' key={review.review_id}>
-              <Link to='/reviews/:review_id'>
+              <Link to={`/reviews/${review.review_id}`}>
                 <h3>{review.title}</h3>
               </Link>
-              <Link to='/reviews/:review_id'>
+              <Link to={`/reviews/${review.review_id}`}>
               <img src={review.review_img_url} alt={review.title} />
               </Link>
               <h4>Review : Click here to read</h4>
```
Comment (mallionaire) [inline src/components/ReviewsList.jsx]: Very misleading heading - the clickable link is the image or the title, but not this h4

**Review by mallionaire — CHANGES_REQUESTED** [review]
Very good start. I've left a couple of comments below, and there are a few other things I'd like you to look at. 
You will need to add some functionality that allows you to display when a review_id is not found. eg, change your url to `reviews/1000` to see what I mean. 
You also appear to have some stale state in your categories selector, when you select a category then click back to "reviews" you re-fetch all reviews but the category is still displayed in the drop down box - think about a way you can clear this selection :D

**Review by Rayhaan2k9 — APPROVED** [review]
Nice work!

**Pull request merged**